### ISSUE TITLE
"Buß- und Bettag" missing for public holiday in Saxony

### ISSUE THREAD

**Issue by jens-maus** [Pix---/ioBroker.feiertage#9]
In Saxony there is still the "Buß- und Bettag" which is a public holiday and will be on the 16th of November in 2016. This holiday is unfortunately missing in the ioBroker.feiertage addon.

**Comment by Pix---** [comment]
Hello jens-maus,
many thanks for your contribution. A brief check shows up a problem with [travis ci](https://travis-ci.org/Pix---/ioBroker.feiertage/builds/172623560)
I look forward to merging your code.

**Comment by jens-maus** [comment]
After having reviewed the travis error I think the issue is not within ioBroker.feiertage but due to some changes in ioBroker.js-controller:

```
/home/travis/build/Pix---/ioBroker.feiertage/node_modules/iobroker.js-controller/lib/tools.js:150
            password(getAppName()).hash(null, null, function (err, res) {
            ^
ReferenceError: password is not defined
    at /home/travis/build/Pix---/ioBroker.feiertage/node_modules/iobroker.js-controller/lib/tools.js:150:13
    at null._onTimeout (/home/travis/build/Pix---/ioBroker.feiertage/node_modules/iobroker.js-controller/lib/objects/objectsInMemServer.js:2068:21)
    at Timer.listOnTimeout (timers.js:119:15)
```

**Comment by jens-maus** [comment]
Finally implemented in version 0.3.3
